app/routes.py

```diff
@@ -1,2 +1,292 @@
-from flask import Blueprint
+from flask.wrappers import Response
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import Blueprint, jsonify, make_response, request
+from datetime import datetime, timezone
+import os
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
 
+#create the blueprint for the endpoints
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+#Create a Task: Valid Task With null completed_at
+@tasks_bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+
+    if "title" not in request_body or "description" not in request_body or "completed_at" not in request_body:
+        return make_response({"details": "Invalid data"}, 400)
+    
+    new_task = set_task(request_body)
+    db.session.add(new_task)
+    db.session.commit()
+
+    response = {}
+    response['task'] = format_task_dictionary(new_task)
+    return make_response(response, 201)
+
+###################
+#Helper functions for tasks
+def set_task(request_body):
+    new_task = Task(title=request_body["title"],
+                description=request_body["description"])
+    
+    if request_body["completed_at"] is not None:
+        new_task.completed_at=request_body["completed_at"]
+    else:
+        new_task.completed_at = None
+    
+    return new_task
+
+def format_task_dictionary(new_task):
```

Comment:
This function, as I remember you mentioning, would work better as an instance method of the `Task` class.